packages/location-field/src/index.js

```diff
@@ -359,9 +359,13 @@ class LocationField extends Component {
     let operatorIcon;
     // Operator only exists on transit features
     const featureIdComponents = source === "transit" && id.split("::");
-    if (featureIdComponents.length > 0) {
+    if (
+      featureIdComponents.length > 0 &&
+      featureIdComponents[1] &&
+      featureIdComponents[1].length > 0
```

Comment:
We have optional chaining on, so I think we can do this instead!
```suggestion
     featureIdComponents.length > 0 && featureIdComponents?.[1].length > 0
```